config/README.md

```diff
@@ -1,102 +1,103 @@
-# Multi-region cockroachdb setup
+# Deploying a DSS
+
+## Preface
+
+This doc provides a well-lit path for deploying the DSS and its dependencies (
+namely CockroachDB) on Kubernetes. This is not a requirement, and a DSS node
+can join a cluster as long as it meets the following requirements:
+
+*   Every CockroachDB node must advertise a unique and routable address.
+    *   It is preferred to use domain names with unique prefixes and homogenous
+        suffixes, ie: 0.c.dss.interussplatform.com.
+    *   This allows wildcard usage in the CRDB certificates.
+*   Every DSS node should run a minimum of 3 CockroachDB nodes.
+*   At least 3 CockroacbDB addresses must be shared with all participants.
+    *   If not using the recommended hostname prefix above, every cockroachDB
+        hostname must be shared with every participant.
+*   Every DSS node must supply and share their CockroachDB public certificate
+*   All CockroachDB nodes must be run in fully secure mode.
```

Comment:
Can we be more specific here to make it easier to verify (non-)compliance?  Is there a single specific flag, for instance, that implementers must include or exclude?

Reply:
done